### PR TITLE
house keeping for profile v2 cis v2

### PR DESCRIPTION
**Pull request by andrewkrug**
I deleted most of what's here and will strategically be pulling in concepts we want to keep from cisv1.  

Mostly though this is blowing up the monolithic library into multiple libraries and abstracting business logic from the serverless functions.

Better interfaces.  More mocks.  Two faker libraries.

### CODE REVIEW

File: docs/Event.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ Or how to send an identity event that CIS will correctly validate and understand
 
 ## Publisher key and CIS stream access
```
Comment (gdestuynder) [inline docs/Event.md]: i suspect we'll have a few more passes at this doc but wfm for now

Comment (gdestuynder) [reply docs/Event.md]: (in particular we should just link to an example)

File: python-modules/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# Mozilla Change Integration Python Modules
+
+Mozilla IAM uses a variety of python modules to achieve integration
+across the stack.  In version 1 of the change integration service one library
+formerly known as `cis` held all this functionality.  In v2 of the service separation of concerns requires a more loosely coupled approach.
```
Comment (gdestuynder) [inline python-modules/README.md]: would be good to write "why" instead

File: python-modules/README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+## Modules in this directory.  
+
+**cis-common**
+
+Code that is common to all additional libs.  Configuration management, authorization layers, etc.
+
+_User Stories_
+
+As a publisher I must exchange a CLIENT_ID and CLIENT_SECRET with auth0 for a bearer token.  This grants the ability to publish changes.
```
Comment (gdestuynder) [inline python-modules/README.md]: `access token`
i know the text in the header says bearer, but it's an access token =)

File: python-modules/README.md (hunk 2 above)
Comment (gdestuynder) [reply python-modules/README.md]: define "publisher" and "processor" here

File: docs/images/CIS-AWS-Stencils.xml [hunk 4]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gdestuynder) [inline docs/images/CIS-AWS-Stencils.xml]: is there a render of this anywhere?

Comment (andrewkrug) [reply docs/images/CIS-AWS-Stencils.xml]: Yeah it's the DOC

**Comment by gdestuynder** [comment]
added a commit with nits instead of comments to save time hopefully

**Pull request merged**